rest-service/manager_rest/rest/endpoint_mapper.py

```diff
@@ -28,6 +28,7 @@ def setup_resources(api):
         'Blueprints': 'blueprints',
         'BlueprintsId': 'blueprints/<string:blueprint_id>',
         'BlueprintsIdArchive': 'blueprints/<string:blueprint_id>/archive',
+        'BlueprintsGlobal': 'blueprints/<string:blueprint_id>/global',
```

Comment:
if we only allow set global, and not unset, etc, maybe we should call it setglobal.. :)